lib/Settings/AdminSection.php

```diff
@@ -43,7 +43,7 @@ public function getName(): string {
 	 * @return string
 	 */
 	public function getIcon(): string {
-		return $this->urlgen->imagePath(Application::APP_ID, 'app.svg');
```

Comment:
Does this actually change anything? Is -dark some kind of magic string?

Reply:
The magic happens with the "fill" attribute on the `<svg>` tag. I always keep a dark and a light version just in case. You can get rid if the light one if you want.

Reply:
So, what does this fix? getIcon() used to return app.svg which was dark. Now it returns app-dark.svg which is dark.

Reply:
app.svg is used in the app list (app settings)
app-dark.svg must be used as the admin settings section icon.

Check those 2 places with dark and light theme and you'll see the icon color was wrong before and is ok now :grin: .

Reply:
Oof, magic file names... Thanks for this, then :blue_heart: